src/Shift-ClassInstaller.package/ShiftClassInstaller.class/class/validateClassName..st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+validateClassName: aString 
+	ShiftClassBuilder new name: aString.
```

Comment:
maybe some new class (ClassNameValidator) used in ShiftClassBuilder too would be better. This behavior is too cryptic.